app/src/renderer/components/governance/ModalPropose.vue

```diff
@@ -200,83 +216,42 @@ export default {
     close() {
       this.$emit(`update:showModalPropose`, false)
     },
-    togglePassword() {
-      this.showPassword = !this.showPassword
+    validateForm() {
+      this.$v.$touch()
+
+      if (!this.$v.$invalid) {
+        this.submitForm()
```

Comment:
I think we should await this promise (+ make the event handler async) unhandled promises sometimes will throw warnings in the console (and when the linter will be on 🎉 won't let them pass in theory)